charts/spire/charts/spire-server/values.yaml

```diff
@@ -185,12 +185,19 @@ upstreamAuthority:
     enabled: false
     rbac:
       create: true
-    issuer_name: "spire-ca"
+    # -- Defaults to the release name, override if CA is provided outside of the chart
+    issuer_name: ""
     issuer_kind: "Issuer"
     issuer_group: "cert-manager.io"
     # -- Specify to use a namespace other then the one the chart is installed into
     namespace: ""
     kube_config_file: ""
+
+    # -- Creates a Cert-manager CA
+    createCA: false
```

Comment:
Should we consider defaulting the createCA function to true so users don't have to bring their own by default?

Reply:
That also requires them to have cert-manager installed. So I prefer to have it on false. to not mandate them into dependencies they don't want.

Reply:
@marcofranssen we're already there I think. To get that setting the user already had to set:
```
upstreamAuthority.certManager.enabled=false
```
and its got to be installed with crds for that plugin to work.